python-avd/pyavd/validation_result.py

```diff
@@ -1,7 +1,12 @@
 # Copyright (c) 2023-2024 Arista Networks, Inc.
 # Use of this source code is governed by the Apache License 2.0
 # that can be found in the LICENSE file.
-from .vendor.errors import AvdDeprecationWarning, AvdValidationError
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from .vendor.errors import AvdDeprecationWarning, AvdValidationError  # pylint: disable=import-outside-toplevel
```

Comment:
Should not be needed in TYPE_CHECKING?
```suggestion
    from .vendor.errors import AvdDeprecationWarning, AvdValidationError
```